tests/test_octodns_provider_ns1.py

```diff
@@ -1199,47 +1199,47 @@ def test_extra_changes(self, monitors_for_mock):
         self.assertFalse(extra)
         monitors_for_mock.assert_not_called()
 
-    def test_has_dynamic(self):
-        provider = Ns1Provider('test', 'api-key')
-
-        desired = Zone('unit.tests.', [])
+    DESIRED = Zone('unit.tests.', [])
 
-        simple = Record.new(desired, 'sim', {
-            'ttl': 33,
-            'type': 'A',
-            'value': '1.2.3.4',
-        })
+    SIMPLE = Record.new(DESIRED, 'sim', {
+        'ttl': 33,
+        'type': 'A',
+        'value': '1.2.3.4',
+    })
 
-        # Dynamic record, inspectable
-        dynamic = Record.new(desired, 'dyn', {
-            'dynamic': {
-                'pools': {
-                    'iad': {
-                        'values': [{
-                            'value': '1.2.3.4',
-                        }],
-                    },
+    # Dynamic record, inspectable
+    DYNAMIC = Record.new(DESIRED, 'dyn', {
+        'dynamic': {
+            'pools': {
+                'iad': {
+                    'values': [{
+                        'value': '1.2.3.4',
+                    }],
                 },
-                'rules': [{
-                    'pool': 'iad',
-                }],
-            },
-            'octodns': {
-                'healthcheck': {
-                    'host': 'send.me',
-                    'path': '/_ping',
-                    'port': 80,
-                    'protocol': 'HTTP',
-                }
             },
-            'ttl': 32,
-            'type': 'A',
-            'value': '1.2.3.4',
-            'meta': {},
-        })
+            'rules': [{
+                'pool': 'iad',
+            }],
+        },
+        'octodns': {
+            'healthcheck': {
+                'host': 'send.me',
+                'path': '/_ping',
+                'port': 80,
+                'protocol': 'HTTP',
+            }
+        },
+        'ttl': 32,
+        'type': 'A',
+        'value': '1.2.3.4',
+        'meta': {},
+    })
+
+    def test_has_dynamic(self):
```

Comment:
Didn't look in detail, but hoping this is not "update"d in multiple tests in the same run. Assuming we need a pristine copy of these records for each test and that behavior is not changed by moving them as instance variables.